### PR TITLE
Correct schema for mapped object with other keys

### PR DESCRIPTION
**Pull request by netanel-mce**
what is here:

- fix issue: https://github.com/YousefED/typescript-json-schema/issues/480#issuecomment-1687743336 
- auto test
- update docs

### CODE REVIEW

**Review by domoritz — APPROVED** [review]

**Comment by netanel-mce** [comment]
@domoritz I updated the version to 0.60.0, please release a new version with this PR.

**Comment by domoritz** [comment]
Please remove the version bump. I will do that when I merge as I will add a tag as well then.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "typescript-json-schema",
-  "version": "0.59.0",
+  "version": "0.60.0",
```
Comment (domoritz) [inline package.json:3]:
```suggestion
  "version": "0.59.0",
```

**Comment by domoritz** [comment]
Thank you. Released as 0.60

**Pull request merged**